documentation/modules/configuring/con-config-node-pools.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='config-node-pools-{context}']
+= (Preview) Configuring node pools
+
+[role="_abstract"]
+Update the `spec` properties of the `KafkaNodePool` custom resource to configure a node pool deployment.
+
+NOTE: The node pools feature is available as a preview. Node pools are not enabled by default, so you must xref:ref-operator-kafka-node-pools-feature-gate-{context}[enable the `KafkaNodePools` feature gate] before using them. 
+
+A node pool refers to a distinct group of Kafka nodes within a Kafka cluster.
+Each pool has its own unique configuration, which includes mandatory settings for the number of replicas, roles, and storage allocation.
+
+Optionally, you can also specify values for the following properties:
+
+* `resources` to specify memory and cpu requests and limits
+* `template` to specify custom configuration for pods and other Kubernetes resources
+* `jvmOptions` to specify custom JVM configuration for heap size, runtime and other options 
+
+The `Kafka` resource represents the configuration for all nodes in the Kafka cluster.
+The `KafkaNodePool` resource represents the configuration for nodes only in the node pool.
+Configuration options not specified in the `KafkaNodePool` resource are inherited from the `Kafka` resource.
+Configuration specified in the `KafkaNodePool` resource takes precedence if set in both resources
+For example, if `jvmOptions` are specified in the node pool configuration and Kafka configuration, the node pool configuration is used.
+
+Node pools can be used with Kafka clusters that operate in KRaft mode (using Kafka Raft metadata) or use ZooKeeper for cluster management.
+If you are using KRaft mode, you can specify roles for all nodes in the node pool to operate as brokers, controllers, or both.
+If you are using ZooKeeper, nodes must be set as brokers only.
+
+IMPORTANT: **KRaft mode is not ready for production in Apache Kafka or in Strimzi.**
+
+For a deeper understanding of the node pool configuration options, refer to the link:{BookURLConfiguring}[Strimzi Custom Resource API Reference^].
+
+NOTE: While the `KafkaNodePools` feature gate that enables node pools is in alpha phase, replica and storage configuration properties in the `KafkaNodePool` resource must also be present in the `Kafka` resource. The configuration in the `Kafka` resource is ignored when node pools are used. Similarly, ZooKeeper configuration properties must also be present in the `Kafka` resource when using KRaft mode. These properties are also ignored.
+
+.Example configuration for a node pool in a cluster using KRaft mode 
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaNodePoolApiVersion}
+kind: KafkaNodePool
+metadata:
+  name: dual-role # <1>
```

Comment:
```suggestion
  name: kraft-dual-role # <1>
```

Reply:
Suggested this name change just to make it clear that dual role is about a KRaft example.